nixos/modules/security/auditd.nix

```diff
@@ -6,6 +6,10 @@ with lib;
   options.security.auditd.enable = mkEnableOption "the Linux Audit daemon";
 
   config = mkIf config.security.auditd.enable {
+    boot.kernelParams = [ "audit=1" ];
+
+    environment.systemPackages = [ pkgs.audit ];
```

Comment:
I'm of the opinion we should avoid installing packages because it is not possible to opt-out of it.

Reply:
I see, but isn't one opts in when they enable `auditd` module? Otherwise it's difficult to decide when we should stop with pre-setting options (for example - should we set `audit=1` here?). My usual rule-of-thumb is "if the daemon needs to be controlled by a user we should install the package".

Reply:
@FRidh does that sound reasonable to you?

Reply:
I don't know. Its hard to decide where to draw the line. I agree that at least a user needs it, but should all. Maybe because we do not have a general way to deal with such issues we should just ignore it and go ahead with it.

Reply:
You raises a good point in that there is no general consensus when it is acceptable to add a package to `environment.systemPackages` in a module. It would be nice to have some guidelines in the NixOS manual, after a decision has been adequately discussed and agreed upon. As you mention, though, this may not be the time and PR to draw the line in the sand... :smile: